include/sundials/sundials_linearsolver.h

```diff
@@ -53,164 +53,161 @@
 #ifndef _SUNLINEARSOLVER_H
 #define _SUNLINEARSOLVER_H
 
-#include <sundials/sundials_types.h>
 #include <sundials/sundials_iterative.h>
 #include <sundials/sundials_matrix.h>
 #include <sundials/sundials_nvector.h>
+#include <sundials/sundials_types.h>
 
-#ifdef __cplusplus  /* wrapper to enable C++ usage */
-extern "C" {
+#ifdef __cplusplus /* wrapper to enable C++ usage */
+extern "C"
+{
 #endif
```

Comment:
I don't think the formatter should indent everything within the `extern "C"` scope.

Reply:
Yes, I fixed this in the Kokkos vector branch, but I can apply it here if you prefer.

Reply:
Fixed.